source/includes/mongodb-compatibility-table-kotlin-sync.rst

```diff
@@ -0,0 +1,96 @@
+.. sharedinclude:: dbx/compatibility-table-legend.rst
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Kotlin Sync Driver Version
+     - MongoDB 8.0
+     - MongoDB 7.0
+     - MongoDB 6.1
+     - MongoDB 6.0
+     - MongoDB 5.0
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+     - MongoDB 3.4
+     - MongoDB 3.2
+     - MongoDB 3.0
+     - MongoDB 2.6
```

Comment:
S: I think it's safe to take these off the chart because they're EOL, and since MDB 3.4 is incompatible users can infer that these are too

Reply:
Updated!